tox.ini

```diff
@@ -13,6 +13,7 @@ envlist =
     pyroma
     flake8
     unittests
+    pytest
```

Comment:
I think this line should not be necessary. the "env"s specified here correspond to the `testenv:SOMETHING` environments.